R/prep.tree.R

```diff
@@ -13,10 +14,10 @@ prep.tree <- function(
     get.root.node(tree.df);
 
     if ('angle' %in% colnames(tree.df)) {
-        message(paste(
-            'Overriding branch angles will be supported in a future version.',
-            'The angle column will not be used.'
-            ));
+        tree.df$angle <- as.numeric(tree.df$angle);
+        if (angle.unit == 'degrees') {
```

Comment:
Is `degrees` the only option for `angle.unit`? would it make more sense to have a binary option instead? like `use.radian = TRUE/FALSE` other wise it seems like there are more options

Reply:
Yeah, right now it's just binary (degrees or radians), and unlikely to extend beyond that. (I can't really think of any other options worth including). Maybe `radian.angles = FALSE` is clearer? In that case, you'd only see `SRCGrob(radian.angles = TRUE)` in cases where they'd override.

There are a lot of boolean toggles like this in BPG, but that hasn't been the most intuitive for me as a BPG user. However, it's very possible that I'm in the minority on this. Do you personally mind the "BPG style" toggles?

Reply:
I think in this case the boolean toggles makes more sense. I can't think off the top of my head which BPG boolean setting particularly irks me. I'm assuming the `radian.angles=TRUE` will be the default?

Reply:
> What would `SRCGrob(radian.angles = TRUE)` look like without specifying anything else?

This would behave a lot like BPG, where the toggle just wouldn't affect anything if the angles aren't specified in the `angle` column.